### PR TITLE
Extend scs-0003: more context and process (resolves issues/#343)

### PR DESCRIPTION
**Pull request by mbuechse**
- added lots of context in the introduction
- introduced the distinction between "certificate" and "certificate type"
- improved misleading wording in the motivation
- rectified incorrect section levels
- added process description (governance)

### CODE REVIEW

File: Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md [hunk 1]
```diff
@@ -10,19 +10,47 @@ track: Global
...
-certification levels and the mandatory standards therein.
+SCS plans to offer six types of certificates, which are sorted into two dimensions:
+
+1. _certification level_, of which there are three:
+   - SCS-compatible
+   - SCS-open
+   - SCS-sovereign
+2. _cloud layer_, of which there are two:
+   - infastructure as a service (IaaS)
+   - Kubernetes as a service (KaaS)
```
Comment (maxwolfs) [inline Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md:21]: I think we need a decision on how to call this layer and implement it everywhere. The team is currently called `Team Container` and often CaaS is preferred over KaaS, as it doesn't only hold Kubernetes.

Comment (mbuechse) [reply Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md:21]: Yes, let's talk about that in the SIG meeting on 2023-08-24. As of right now, `kaas` is the term used in the yaml and this document, and I stuck to it.

**Review by garloff — APPROVED** [review]
/lgtm
Thanks @mbuechse

**Comment by mbuechse** [comment]
@alexander-diab This PR is good to be merged; Kurt is satisfied, apparently highly so. Do you have any objections?

File: Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md [hunk 2]
```diff
@@ -10,34 +10,62 @@ track: Global
...
+
+This decision record describes two main points:
+
+1. How we denote our certificate types by means of a YAML file.
+2. Our process for constructing and progressing the certificate types.
 
 ## Motivation
 
-This decision record has three main objectives:
+This decision record establishes a mechanism (by means of the YAML file) with the following three main objectives:
```
Comment (anjastrunk) [inline Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md:47]: Isn't this of type `Procedural` and not of `Decision Record`?

```suggestion
This standard establishes a mechanism (by means of the YAML file) with the following three main objectives:
```

Comment (mbuechse) [reply Standards/scs-0003-v1-sovereign-cloud-standards-yaml.md:47]: The verbiage in question didn't stem from me. Anyway, following your argumentation, we should change "decision record" into "procedural", right? However, I suppose "decision record" could be construed as a kind of general term (a superset) that includes Standards, Procedurals, and any other document. From this point of view, every standard is a decision record, but not every decision record is a standard. If we expressly designate a document as "decision record" in the header, then this mostly means that it doesn't fit into any of the subcategories. Just one way of looking at it.

**Comment by anjastrunk** [comment]
Regarding yaml file:

Do we have **one** yaml file with **all** layers and levels at the end, or **one** yaml file per level and layer, such as SCS-compatible-IaaS.yaml for layer IaaS and level "SCS-compatible"

I prefer the second option, as one file may be grow over time and gets confusting. We may even think to have one file or each version, such as  SCS-compatible-IaaS-v1.yaml for version 1 of  layer "SCS-compatible" certificate for IaaS.

**Comment by mbuechse** [comment]
> Regarding yaml file:
> 
> Do we have **one** yaml file with **all** layers and levels at the end, or **one** yaml file per level and layer, such as SCS-compatible-IaaS.yaml for layer IaaS and level "SCS-compatible"

Currently, it's one file per level, containing both layers. I would prefer one file per type of certificate, that is to say, one file per combination of level and layer. But this has been an open topic in the SIG for months now.

**Comment by anjastrunk** [comment]
I like your improvements. It makes thing clearer.

One question, which came to may mind: Is there already a standard for receiving a certificate as CSP? I mean, how das a CSP apply for a certificate, how often will a re-certification take place and what are the consequences if certificate rules are violated?

**Comment by mbuechse** [comment]
> One question, which came to may mind: Is there already a standard for receiving a certificate as CSP? I mean, how das a CSP apply for a certificate, how often will a re-certification take place and what are the consequences if certificate rules are violated?

@anjastrunk Something like that is being worked on in https://github.com/SovereignCloudStack/standards/issues/337

**Comment by anjastrunk** [comment]
> > Regarding yaml file:
> > Do we have **one** yaml file with **all** layers and levels at the end, or **one** yaml file per level and layer, such as SCS-compatible-IaaS.yaml for layer IaaS and level "SCS-compatible"
> 
> Currently, it's one file per level, containing both layers. I would prefer one file per type of certificate, that is to say, one file per combination of level and layer. But this has been an open topic in the SIG for months now.
> 
> @anjastrunk see here: https://input.scs.community/2023-scs-sig-standardization#2023-06-29 I'm pretty sure that I had raised the matter even earlier, but it got postponed at least once, and even when it got discussed, it was postponed again.

We should but this on next SIG' agenda and make a decision!

**Review by anjastrunk — APPROVED** [review]

**Comment by anjastrunk** [comment]
> > One question, which came to may mind: Is there already a standard for receiving a certificate as CSP? I mean, how das a CSP apply for a certificate, how often will a re-certification take place and what are the consequences if certificate rules are violated?
> 
> @anjastrunk Something like that is being worked on in #337

Yes. Thanks.

**Comment by mbuechse** [comment]
> We should put this on next SIG' agenda and make a decision!

I prepared the agenda accordingly.

**Pull request merged**